LICENSE

```diff
@@ -1,5 +1,6 @@
-MIT License
+The MIT License (MIT)
 
+Open3D: www.open3d.org
```

Comment:
Just question: do we need to change license file?

Reply:
sorry about the confusion.

We used to have two license files, one in open3d/ and one in open3d/src.
This is my proposal to keep just one. It's the license we used to have in open3d/src/ because it had one additional line: `Open3D: www.open3d.org`

Any other changes are just wraparound differences.